### PR TITLE
Add integ test for data message

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
Add an integration test for data message.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes
2. How did you test these changes?
Manually run the integration test.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,6 +56,14 @@ jobs:
         - source script/travis-awscli-installation
         - integration/js/script/install-kite
       script: npm run test:integration-content-share
+    #Integ test content share
```
Comment (aburnson) [inline .travis.yml:59]: Typo in comment, but also I don't think we need these comments as they are redundant

File: integration/js/checks/DataMessageCheck.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  stepDescription() {
+    return 'Check data message: ' + this.message;
+  }
+
+  async run() {
+    try {
+      const result = await this.page.checkDataMessageExist(this.message);
+      if (!result) {
+        this.testReporter.textAttachment(this.report, 'Data message', result, 'plain');
+        throw new KiteTestError(Status.FAILED, `Mesasge ${this.message} not found`);
```
Comment (aburnson) [inline integration/js/checks/DataMessageCheck.js:24]: Typo: `s/Mesasge/Message`

File: integration/js/pages/AppPage.js [hunk 3]
```diff
@@ -493,6 +494,17 @@ class AppPage {
     return expectedState
   }
 
+  async sendDataMessage(message) {
+    const dataMessageSendInput = await this.driver.findElement(elements.dataMessageSendInput);
+    await dataMessageSendInput.clear();
+    await dataMessageSendInput.sendKeys(message);
+    await dataMessageSendInput.sendKeys(Key.ENTER);
+  }
```
Comment (aburnson) [inline integration/js/pages/AppPage.js:502]: Should we add a little bit of a wait (say 1 second) to give the message time to be delivered?

File: .travis.yml (hunk 1 above)
Comment (ltrung) [reply .travis.yml]: Fixed

File: integration/js/checks/DataMessageCheck.js (hunk 2 above)
Comment (ltrung) [reply integration/js/checks/DataMessageCheck.js]: Fixed.

File: integration/js/pages/AppPage.js (hunk 3 above)
Comment (ltrung) [reply integration/js/pages/AppPage.js:502]: I added one sec in the test before validating that the message is sent to the other attendee.

**Pull request merged**